edb/server/compiler/ddl.py

```diff
@@ -1169,6 +1184,53 @@ def _reset_schema(
     )
 
 
+_FEATURE_NAMES: dict[type[s_obj.Object], str] = {
+    s_policies.AccessPolicy: 'policy',
+    s_triggers.Trigger: 'trigger',
+    s_rewrites.Rewrite: 'rewrite',
+    s_globals.Global: 'global',
+    s_expraliases.Alias: 'alias',
+    s_func.Function: 'function',
```

Comment:
Not necessarily for this PR, but maybe we can add:

* link properties
* more than one base type on types
* annotations
* indexes (i'd like to see how often people define them!)
* constraints (ditto)
* computed properties (ditto)
* computed links
* multi properties
* enums

Reply:
@zackelan I can't tell if I'm missing something here: is there are major difference between what you are suggesting we do and what I'm doing here?

Reply:
@1st1 I added all those (a third or so were already there)

Reply:
nothing in particular - just a +1 to Yury's suggestion of adding metrics like Oprah giving away cars